package.json

```diff
@@ -47,10 +47,7 @@
     "eslint-plugin-react-hooks": "^4.2.0",
     "jest": "^26.6.3",
     "jest-extended": "^0.11.5",
-    "metro-react-native-babel-preset": "^0.66.0",
```

Comment:
Hi @pdpino , I couldn't run the example app, because the react-native is also installed in the root folder, so I have to remove these dependencies in this PR. Did you get the same issue?

Reply:
> Did you get the same issue?

No, I have no problem running the example app. I run:
```
cd example
yarn
yarn android
yarn start
```

I think the packages installed in the main folder have nothing to do with the packages installed in the example app. What error are you getting?

Reply:
there'll be two different react native versions, one in root, one in example. Can you try to delete the node_modules in both root and example folders, then run yarn install again? After that, when you run the example app, it will show the error.

Reply:
> there'll be two different react native versions, one in root, one in example

I see. I tried what you suggested and I got an error, something like this:

```
bundling failed: ambiguous resolution: module `/.../react-native-week-view/example/index.js` tries to require `react-native`, but there are several files providing this module. You can delete or fix them
```

I think the solution would be to use the same versions in the example app and in the devDependencies from the root folder, so we can run the example app and run the tests. I haven't tested it though

Reply:
It doesn't work, even we use the same version. It's not the version problem, the example app couldn't understand the react native from root folder.

Reply:
That's too bad, there must be a way to run the tests and run the app. Maybe you can merge this PR for now to fix the exampl app, and we can fix the tests in another PR?

Reply:
Thanks @pdpino , then I'll merge this PR first, we can find another solution after it.